viabel/stan_models/weighted_lin_regression_sgd.stan

```diff
@@ -1,4 +1,8 @@
 data {
+    // weighted regression model used to estimate constants that required to 
```

Comment:
This comment should go above the data block since it's explaining what the whole model is for